tests/part.rs

```diff
@@ -61,7 +61,10 @@ fn part_implicit_init_call() {
                 object
                     .get_property_value(&Identifier(Refer::none("b".into())))
                     .expect("Property `b`"),
-                &Value::Scalar(value)
+                &Value::Quantity(Quantity::new(
+                    value,
+                    microcad_lang::ty::QuantityType::Scalar
```

Comment:
use `use ty::*` instead

```suggestion
                    QuantityType::Scalar
```